### DISCUSSION TITLE
Opening up Discussions

### DISCUSSION THREAD

**Discussion by allmarkedup** [lookbook-hq/lookbook#100] [Announcements]
Hey all 👋

I'm opening up discussions here as a place to ask more general questions about Lookbook, get help with things that aren't clear in the documentation (probably lots), showcase any publicly-accessible ViewComponent/Lookbook implementations that people want to share and absolutely else Lookbook-related that people want to discuss.

**Any bugs should still be opened as an issue** but feature requests and suggestions should probably start life off here so they can be discussed before opening as a proper issue to track their implementation progress, if it gets to that.

And if you feel like just introducing yourself here as a Lookbook user then please feel free! :-)

**Comment by tripptuttle** [comment]
Love this @allmarkedup! And thank you for bringing Lookbook to us!

**Reply by allmarkedup** [reply]
My pleasure @tripptuttle - I'm just pleased that people find it useful. And thanks for your positivity!
